dan-stineback/test/error_services_test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const angular = require('angular');
+require('angular-mocks');
+require('../app/js/client');
+
+describe('error service tests', function() {
+  let errorService;
+  beforeEach(() => {
+    angular.mock.module('FirstApp');
+    angular.mock.inject(function(ErrorService){
+      errorService = ErrorService;
+    });
+  });
+  it('should have test getErrors', () => {
+    expect(typeof errorService.getErrors).toBe('function');
+  });
+  it('should test logErorr', () => {
+    expect(typeof errorService.logErorr).toBe('function');
+  });
+  it('should test errors', () => {
+    expect(Array.isArray(errorService.getErrors())).toBe(true);
```

Comment:
Would be good to test logging an error and retrieving result, like 
 errorService.logError('test error')({});
    let errorArray = errorService.getErrors();
    expect(errorArray[0]).toBe('test error');